### PR TITLE
Fixes 3619: Update console & classlib project templates to opt-in to ImplicitUsings feature

### PR DESCRIPTION
**Pull request by DavidKarlas**
### Problem
SDK is switching from `<DisableImplicitNamespaceImports>` switching to `<ImplicitUsings>`, so should templates... See https://github.com/dotnet/templating/issues/3619 for more details.

### Solution
Changed text in .csproj and disabled `dotnet build` part of tests until SDK does their side of change, tracked under https://github.com/dotnet/templating/issues/3634

### Checks:
- [x] Added unit tests
- [x] Added `#nullable enable` to all the modified files [?](https://github.com/dotnet/templating/wiki/Contributing#coding-style)

### CODE REVIEW

**Review by bekir-ozturk — APPROVED** [review]

**Comment by DavidKarlas** [comment]
Turns out there are 3 other sets of tests that are accepting --framework and --langVersion parameters... Hence those are failing too... Lets wait for SDK change for few days...

**Comment by pranavkm** [comment]
@DavidKarlas can you elaborate on what SDK changes you're currently waiting for?

**Comment by DavidKarlas** [comment]
@pranavkm if you try to compile project that has:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <LangVersion>9.0</LangVersion>
  </PropertyGroup>
</Project>
```
It fails with:
```
/Users/davidkarlas/tmp/a12/obj/Debug/net6.0/a12.ImplicitNamespaceImports.cs(2,1): error CS8773: Feature 'global using directive' is not available in C# 9.0. Please use language version 10.0 or greater. [/Users/davidkarlas/tmp/a12/a12.csproj]
```
My understanding is... Reason for this is that current SDK expect me to do `<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>`, but going forward above .csproj will just work, because it doesn't have `<ImplicitImports>enable</ImplicitImports>`...

File: template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Company.ClassLibrary1.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.ClassLibrary1</RootNamespace>
     <LangVersion Condition="'$(langVersion)' != ''">$(ProjectLanguageVersion)</LangVersion>
-    <DisableImplicitNamespaceImports Condition="'$(csharpFeature_ImplicitUsings)' != 'true'">true</DisableImplicitNamespaceImports>
+    <ImplicitUsings Condition="'$(csharpFeature_ImplicitUsings)' == 'true'">enable</ImplicitUsings>
```
Comment (pranavkm) [inline template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Company.ClassLibrary1.csproj:8]:
```suggestion
     <DisableImplicitNamespaceImports Condition="'$(csharpFeature_ImplicitUsings)' != 'true'">true</DisableImplicitNamespaceImports>
    <ImplicitUsings Condition="'$(csharpFeature_ImplicitUsings)' == 'true'">enable</ImplicitUsings>
```

File: template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApplication-CSharp/Company.ConsoleApplication1.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.ConsoleApplication1</RootNamespace>
     <LangVersion Condition="'$(langVersion)' != ''">$(ProjectLanguageVersion)</LangVersion>
-    <DisableImplicitNamespaceImports Condition="'$(csharpFeature_ImplicitUsings)' != 'true'">true</DisableImplicitNamespaceImports>
+    <ImplicitUsings Condition="'$(csharpFeature_ImplicitUsings)' == 'true'">enable</ImplicitUsings>
```
Comment (pranavkm) [inline template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApplication-CSharp/Company.ConsoleApplication1.csproj:9]:
```suggestion
    <DisableImplicitNamespaceImports Condition="'$(csharpFeature_ImplicitUsings)' != 'true'">true</DisableImplicitNamespaceImports>
    <ImplicitUsings Condition="'$(csharpFeature_ImplicitUsings)' == 'true'">enable</ImplicitUsings>
```

**Review by pranavkm — APPROVED** [review]

**Pull request merged**